test/representers/api/paged_collection_representer_test.rb

```diff
@@ -6,22 +6,17 @@
 
 describe Api::PagedCollectionRepresenter do
 
-  let(:items)            { (0..25).collect{|t| TestObject.new("test #{t}", true) } }
-  let(:paged_items)      { Kaminari.paginate_array(items).page(1).per(10) }
-  let(:request)          { OpenStruct.new(params: {"page"=>"1", "action"=>"index", "api_version"=>"v1", "controller"=>"api/test_objects", "format"=>"json"}) }
+  let(:items) { (0..25).collect{|t| TestObject.new("test #{t}", true) } }
```

Comment:
Prefer `map` over `collect`.<br>Space missing to the left of {.<br>Space between { and | missing.